src/main/java/qna/domain/Answer.java

```diff
@@ -96,4 +98,15 @@ public void setDeleted(boolean deleted) {
 		this.deleted = deleted;
 	}
 
+	public DeleteHistory delete(User loginUser) throws CannotDeleteException {
+		validateDelete(loginUser);
+		this.setDeleted(true);
```

Comment:
```suggestion
		this.deleted = true;
```
이렇게 표현해도 되지 않을까요? 무의미한 setter는 지양하는게 좋을 것 같습니다

Reply:
네! 감사합니다~